### PR TITLE
perl-gd : fontconfig dep (runtime lib load error)

### PR DESCRIPTION
**Pull request by martin-mann**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

when using perl-gd, the libfontconfig.so lib cannot be found

### CODE REVIEW

**Comment by martin-mann** [comment]
Whatthef... the new libgd package 2.2.3-2 was just compiled and released....

Osx build fails now with:
```
conda.exceptions.PaddingError: Placeholder of length '32' too 
   short in package bioconda::libgd-2.2.3-2.
The package must be rebuilt with conda-build > 2.0
```

@bgruening any idea

File: recipes/perl-gd/meta.yaml [hunk 1]
```diff
@@ -14,15 +14,13 @@ requirements:
     - llvm # [osx]
     - perl-threaded >=5.22.0
     - libgd >=2.2.3
-    - fontconfig # for libfontcontig.so
```
Comment (bgruening) [inline recipes/perl-gd/meta.yaml]: Martin these `#` are not comments.
`#` is part of the template language. So maybe these things are interpreted as some variable?

Comment (martin-mann) [inline recipes/perl-gd/meta.yaml]: I dont think so, but we can try if it fails again. Last fail was most likely due to an ancient dep build...

Comment (bgruening) [inline recipes/perl-gd/meta.yaml]: Ha, good to know!

**Comment by martin-mann** [comment]
The osx prefix problem persidts even with rebuild libgd package...

**Comment by bgruening** [comment]
Let's see ...

**Pull request merged**